src/pages/STG_PanelAddrVerification.page

```diff
@@ -42,152 +42,143 @@
                 } else {
                     j$('[id$=iSettingsHelpText]').html(settingsHelpText[selection]);
                     j$('[id$=Enable_Automatic_Verification__c').prop('checked', true);
-                    
-                    hideUnusedElements();    
+
+                    hideUnusedElements();
                 }
-            }); 
+            });
         });
     </script>
+    <style>
+        div.slds-text-body--small ol {
+            list-style: initial;
+            list-style-type: decimal;
+        }
+    </style>
+    <apex:form id="form" styleClass="slds-m-around--x-large">
```

Comment:
I thought when using SLDS, one needed an outer <div class="slds">, which is what I've used in the pages I've SLDS'ed so far.